.travis.yml

```diff
@@ -13,29 +13,83 @@ python:
   - "3.6-dev"  # 3.6 development branch
   - "3.7-dev"  # 3.7 development branch
 env:
-  - VNUMPY=1.8.2
-  - VNUMPY=1.12.1
-  - VNUMPY=1.16.1
+  - VNUMPY=1.8.2  VSCI=0.21.2
+  - VNUMPY=1.8.2  VSCI=0.20.0
+  - VNUMPY=1.8.2  VSCI=0.18
+  - VNUMPY=1.12.1  VSCI=0.21.2
+  - VNUMPY=1.12.1  VSCI=0.20.0
+  - VNUMPY=1.12.1  VSCI=0.18
+  - VNUMPY=1.16.1  VSCI=0.21.2
+  - VNUMPY=1.16.1  VSCI=0.20.0
+  - VNUMPY=1.16.1  VSCI=0.18
+  
 cache:
   - apt
 matrix:
-# old numpy + new python has problems setting up
+# old packagese + new python has problems setting up
   exclude:
     - python: "3.7-dev"
-      env: VNUMPY=1.8.2
+      env: VNUMPY=1.8.2  VSCI=0.21.2
     - python: "3.7-dev"
-      env: VNUMPY=1.12.1
+      env: VNUMPY=1.8.2  VSCI=0.20.0
+    - python: "3.7-dev"
+      env: VNUMPY=1.8.2  VSCI=0.18
+    - python: "3.7-dev"
+      env: VNUMPY=1.12.1  VSCI=0.21.2
+    - python: "3.7-dev"
+      env: VNUMPY=1.12.1  VSCI=0.20.0
+    - python: "3.7-dev"
+      env: VNUMPY=1.12.1  VSCI=0.18
+    - python: "3.7-dev"
+      env: VNUMPY=1.16.1  VSCI=0.18
+    - python: "3.6-dev"
+      env: VNUMPY=1.8.2  VSCI=0.21.2
+    - python: "3.6-dev"
+      env: VNUMPY=1.8.2  VSCI=0.20.0
     - python: "3.6-dev"
-      env: VNUMPY=1.8.2
+      env: VNUMPY=1.8.2  VSCI=0.18
+    - python: "3.6-dev"
+      env: VNUMPY=1.12.1  VSCI=0.18
+    - python: "3.6-dev"
+      env: VNUMPY=1.16.1  VSCI=0.18
+    - python: "3.6"
+      env: VNUMPY=1.8.2  VSCI=0.21.2
+    - python: "3.6"
+      env: VNUMPY=1.8.2  VSCI=0.20.0
     - python: "3.6"
-      env: VNUMPY=1.8.2
+      env: VNUMPY=1.8.2  VSCI=0.18
+    - python: "3.6"
+      env: VNUMPY=1.12.1  VSCI=0.18
+    - python: "3.6"
+      env: VNUMPY=1.16.1  VSCI=0.18
+    - python: "3.5-dev"
+      env: VNUMPY=1.8.2  VSCI=0.21.2
+    - python: "3.5-dev"
+      env: VNUMPY=1.8.2  VSCI=0.20.0
+    - python: "3.5"
+      env: VNUMPY=1.8.2  VSCI=0.21.2
+    - python: "3.5"
+      env: VNUMPY=1.8.2  VSCI=0.20.0
+    - python: "2.7"
+      env: VNUMPY=1.8.2  VSCI=0.21.2
+    - python: "2.7"
+      env: VNUMPY=1.8.2  VSCI=0.20.0
+    - python: "2.7"
+      env: VNUMPY=1.12.1  VSCI=0.21.2
+    - python: "2.7"
+      env: VNUMPY=1.12.1  VSCI=0.20.0
+    - python: "2.7"
+      env: VNUMPY=1.16.1  VSCI=0.21.2
+    - python: "2.7"
+      env: VNUMPY=1.16.1  VSCI=0.20.0
+      
```

Comment:
Okay this looks good. However to have this constantly stable as new scikits versions keep appearing maybe it would be better to [include](https://docs.travis-ci.com/user/customizing-the-build#explicitly-including-jobs) some specific runs, e.g. Python 2.7 with the latest scikits and numpy known to work. Maybe also the oldest versions known to work so we keep track of loosing backwards compatibility at the bottom.